### ISSUE TITLE
about metrics

### ISSUE THREAD

**Issue by as1113046370** [synlp/R2GenRL#1]
Thank you for sharing this project, but I have some questions about it.
Did the seed is a fixed value, I found that the index fluctuated greatly under the influence of different seeds

**Comment by LangWY** [comment]
Hello, may I ask if you have reproduced the results in the paper? If successful, can you tell me what the learning rate and batch have been adjusted to?

**Comment by Junyiliu0** [comment]
Have reproduced the results in the paper? Can you tell me the detail of training?

**Comment by Junyiliu0** [comment]
> Hello, may I ask if you have reproduced the results in the paper? If successful, can you tell me what the learning rate and batch have been adjusted to?

您好，请问您复现成功了吗？您也是做这个方向的吗？可以交流一下吗？
